### DISCUSSION TITLE
Rubberduckvba version control?

### DISCUSSION THREAD

**Discussion by JakabiTV** [rubberduck-vba/Rubberduck#6109] [Q&A]
Hello, in the "README.md" of Rubberduck I've read the sentence "_Of course there's more! There's tooling to help synchronizing the project with files in a folder **(for source/version control)**, some auto-completion features like self-closing parentheses and quotes;_"

It states that there is also a feature which helps with version control of vba projects, but I wasn't able to find any documentation about that on the github page but neither on the homepage. Can anyone help me out?

**Comment by retailcoder** [comment]
The features in question are explained in details in this article:

https://rubberduckvba.wordpress.com/2020/12/11/sync-project-with-files/
